### ISSUE TITLE
As a Neighbor, I would like to sign up for updates on a specific Project, so that I can stay in the loop on what happens with the Project.

### ISSUE THREAD

**Issue by AmyWipfler** [CityOfBoston/boston.gov-d8#1676]

**Comment by AmyWipfler** [comment]
Sprint 3 DoD: Back-end prototype collection of email addresses and storage on SFDC Project records. Stretch = Research "Email List" button on Salesforce related list.

**Comment by AmyWipfler** [comment]
@padora-cob Did some research into our SFDC options this morning in advance of talking to SFDC Admins about this Thursday. We can either go the Contact Related List route on the Project, or we can mimic the EoI structure and skirt the Contact rules that are baked into SFDC.

For example, [this Parcel](https://boston-dnd.my.salesforce.com/a1jC0000000pd1N), is related to [this Project](https://boston-dnd.my.salesforce.com/a040y00000Z88HY), and has EoI's[ like this one](https://boston-dnd.my.salesforce.com/a1p0y000005IuoL) related to it. There's a custom button on the Parcel record "EoI Email List" that generates a report of all emails so it's easier for a PM to send and email from SFDC or copy and paste emails to their Gmail. I think it would be worth exploring expanding this to meet subscriber needs or building something similar that relates to the Project.

In summary, 3 options:
1. Related Contact List on Project
2. Expanded use of the EoI related list on the Parcel
3. Mimic'ed version of the EoI related list on the Project

**Comment by AmyWipfler** [comment]
Explored this during whiteboarding on 11/23/20. The key need here is more of a notification tool that will be set up to trigger certain types of updates to an email list. Without better understanding what the new update object might be yet and how those updates function, we will circle back to prototyping this post-MVP

**Comment by AmyWipfler** [comment]
Whiteboarding screenshot

![image.png](https://images.zenhubusercontent.com/5f774cd93389cdee6cb34b22/edd58da2-41d2-4812-a6f6-f5e0c8b0c810)
